### PR TITLE
refactor(onboarding,downloader): Remove dl_ prefix on downloaded files

### PR DESCRIPTION
**Pull request by Dr0p42**
Simply remove `dl_` prefix as it does not really make sense. Instead we are creating non overlapping names:
- `notebook_(1).ipynb`
- `notebook_(2).ipynb`
- `notebook_(3).ipynb`
- ...

### CODE REVIEW

**Comment by riderx** [comment]
@Dr0p42 the DL was to prevent user to erase the current file when they click on the download link, you should block the ability to erase file if exist

**Comment by jravenel** [comment]
Or add suffix like "my file (2)ipynb". ?

File: naas/runner/controllers/downloader.py [hunk 1]
```diff
@@ -44,7 +44,7 @@ async def get(self, request):
                     {"id": uid, "type": t_downloader, "status": t_send, "filepath": url}
                 )
                 return json({"status": t_error, "error": str(e), "tb": str(tb)})
-        wp_set_for_open_filebrowser(file_name)
+        # wp_set_for_open_filebrowser(file_name)
```
Comment (Dr0p42) [inline naas/runner/controllers/downloader.py:47]: @riderx is it fine to not use this function? (If you say yes I will remove it as it is the only place where it is used).

Seems like it's not "always" working to create and import a new Jupyter workspace.

**Comment by riderx** [comment]
Good for the number in the name !

File: naas/runner/controllers/downloader.py (hunk 1 above)
Comment (riderx) [reply naas/runner/controllers/downloader.py:47]: for me it's better to try to open it and it work sometime than never doing it no ?
If you think with @jravenel it's better to don't have it for now remove it, but i think you should try to improve

**Pull request merged**